src/resultsutil.h

```diff
@@ -74,6 +74,7 @@ enum class CallbackAction
 {
     ViewCallerCallee = 0x1,
     OpenEditor = 0x2,
+    ViewDisassembly = 0x3
```

Comment:
= 0x4

otherwise `ViewDisassembly` is equal to `ViewCallerCallee | OpenEditor`